ui/app/styles/components/replication-dashboard.scss

```diff
@@ -17,9 +17,20 @@
 
   .selectable-card-container {
     display: grid;
-    grid-template-columns: repeat(auto-fit, minmax(400px, 1fr));
-    grid-template-rows: 1fr;
-    grid-gap: 2rem;
+
+    &.primary {
```

Comment:
woo hoo, this is great. so nice to have the styles for this component all in one spot!